src/dune

```diff
@@ -0,0 +1,8 @@
+(library
+ (name mlmpfr)
+ (public_name mlmpfr)
+ (wrapped false)
+ (modules mpfr)
+ (c_names mlmpfr_stubs)
+ (c_flags -O2 -Wall -Wextra)
+ (c_library_flags -lmpfr))
```

Comment:
You can remove `wrapped`, `name` and `modules` here.

Reply:
I'll try it

Reply:
If I remove wrapped, the testsuite can't find the module Mpfr.

Reply:
OK, I thought libraries with only one modules were not wrapped by default, my bad !